woocommerce/woocommerce-admin
# Load the page controller functions file first to prevent fatal errors when disabling WooCommerce Admin

Fixes #6673 

Load the page controller functions file first to prevent fatal errors when disabling WooCommerce Admin.

Also noted this issue with WC Pay active, but this PR does not address that issue (https://github.com/woocommerce/woocommerce-admin/issues/6703)


### Detailed test instructions:

1. Deactivate WC Admin plugin
1. Checkout this branch and symlink to `woocommerce/packages/woocommerce-admin`
1. Under wp-content create mu-plugins/my-plugin.php
1. Disable WC Admin with add_filter( 'woocommerce_admin_disabled', '__return_true' ); in your my-plugin.php file
1. Add the example plugin linked to by run npm run example -- --ext=add-task
1. Activate the plugin
1. Don’t see the fatal error
